### PR TITLE
Improve test output

### LINKED ISSUE #87: Clean up test output
Generally chatty tests/code should be avoided. This issue suggests getting rid of all the print statements in order to improve the test output.

### PR DESCRIPTION
**Pull request by peterwvj**
Closes #87

### CODE REVIEW

File: lib/utils/HttpHelper.dart [hunk 1]
```diff
@@ -99,7 +94,6 @@ class HttpHelper {
     if (isTestModeActive) {
       var token = 'Basic ' + base64Encode(utf8.encode('off:off'));
       headers.addAll({'authorization': token});
-      print("TEST-MODE");
```
Comment (MohamedFBoussaid) [inline lib/utils/HttpHelper.dart:102]: Maybe we can keep this log to confirm to the user that he is on test mode. 
it is not that big deal, and this is not a blocker to merge.

**Review by MohamedFBoussaid — APPROVED** [review]
Can be merged.

File: lib/utils/HttpHelper.dart (hunk 1 above)
Comment (peterwvj) [reply lib/utils/HttpHelper.dart:102]: Hi @MohamedFBoussaid thanks for taking a look. If this print statement is there it is printed with every test, see
image
Would you be OK with leaving it out?
If it is not there the output looks more pleasant
image

If you still prefer to keep it there then I can add the statement back in.
What do you think?

Comment (MohamedFBoussaid) [reply lib/utils/HttpHelper.dart:102]: In that case let's just delete it, it does not worth it.

Comment (peterwvj) [reply lib/utils/HttpHelper.dart:102]: Will do, thanks!

**Pull request merged**